README.md

```diff
@@ -85,7 +82,7 @@ Publishes SDK to Sonatype repository and releases to Maven Central.
 (Usually you don't need to publish manually because CI/CD workflow publishes automatically.)
 
 ```
-./gradlew :bucketeer:publishReleasePublicationToSonatypeRepository
+./gradlew :bucketeer:publish --no-daemon --no-parallel -PmavenCentralUsername=$USERNAME -PmavenCentralPassword=$PASSWORD
```

Comment:
credential should now be passed like this or declared as environment variables.
I'm gonna change CI config once this gets merged.

https://github.com/vanniktech/gradle-maven-publish-plugin#where-to-upload-to